deploy/deploy-multisig.ts

```diff
@@ -1,47 +1,57 @@
-import { utils, Wallet, Provider, EIP712Signer, types } from "zksync-web3";
-import * as ethers from "ethers";
-import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { utils, Wallet, Provider, EIP712Signer, types } from 'zksync-web3'
+import * as ethers from 'ethers'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
 
 // Put the address of your AA factory
-const AA_FACTORY_ADDRESS = "0x4B5DF730c2e6b28E17013A1485E5d9BC41Efe021";
+const AA_FACTORY_ADDRESS = '<YOUR_FACTORY_ADDRESS>'
 
 export default async function (hre: HardhatRuntimeEnvironment) {
-  const provider = new Provider(hre.config.zkSyncDeploy.zkSyncNetwork);
-  const wallet = (new Wallet("<PRIVATE-KEY>")).connect(provider);
-  const factoryArtifact = await hre.artifacts.readArtifact("AAFactory");
+  const provider = new Provider(hre.config.zkSyncDeploy.zkSyncNetwork)
+  const wallet = new Wallet('<YOUR_PRIVATE_KEY>').connect(provider)
```

Comment:
Let's leave correcting it for some other time, but we usually do use ";" at the end of the line and it is generally considered a good practice

Reply:
yeah, we gotta have a consistent codebase so we should consider sharing VSCode/Prettier/ESLint settings (I'm a single quote guy 😅). Let's merge this now and solve those styling things in the next PRs